manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -148,6 +148,8 @@ The easiest way is to use ``isort``.
 Check the `Python style guide for Plone <https://docs.plone.org/develop/styleguide/python.html#grouping-and-sorting>`_
 for information about the order of imports and an example configuration for ``isort``.
 
+If ``six`` is used in the code,
```

Comment:
I would like to suggest to add a here a link to the package or its documentation, for example: https://six.readthedocs.io/

Doing so is handy because it gives people more background info.
Not everyone knows ``six``

Thanks!

Reply:
The first instance of six in this document is already hyperlinked.